monitor/src/main/java/edu/asu/diging/monitor/core/model/impl/AppHelper.java

```diff
@@ -0,0 +1,23 @@
+package edu.asu.diging.monitor.core.model.impl;
```

Comment:
I just now realize that this is the wrong package for this class. You put it in the model package, which is for domain classes not helpers or managers.